kubernetes/kube-state-metrics-cluster-role.yaml

```diff
@@ -8,6 +8,7 @@ rules:
   - nodes
   - pods
   - resourcequotas
+  - replicationcontrollers
   verbs: ["list", "watch"]
 - apiGroups: ["extensions"]
```

Comment:
Deployments also exists in `apps` now. No need to be this pull, but you're likely to transition at some point.  For some period (if you support both), you can list both in your role